pennylane/beta/plugins/numpy_ops.py

```diff
@@ -0,0 +1,279 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Utility functions and numerical implementations of operations for
+the default.tensor device.
+"""
+import math
+import cmath
```

Comment:
Note changing to using `math` and `cmath` as per (reason for a conflict with master in `default_tensor.py`:
https://github.com/XanaduAI/pennylane/pull/567/files?file-filters%5B%5D=.py